testnet/stacks-node/src/config.rs

```diff
@@ -222,7 +224,7 @@ mod tests {
         assert_eq!(config.burnchain.affirmation_overrides.len(), 1);
         assert_eq!(config.burnchain.affirmation_overrides.get(&0), None);
         assert_eq!(
-            config.burnchain.affirmation_overrides.get(&1),
```

Comment:
same question as above re: changing 1->413

Reply:
Yep, like above -- basically, the existing test supplied an illegal affirmation map / reward cycle number combination.